frontend/smart-kickers-game/src/components/GameResults.js

```diff
@@ -34,7 +38,10 @@ function GameResults({ blueScore, whiteScore }) {
           >
             +
           </Button>
-          White: {whiteScore}
+          <div className="icon-result">
+            <FontAwesomeIcon className="whiteTeamIcon" icon="fa-person" />
```

Comment:
You can also check icons from our company, `sap-icon` package or something like this, I'll catch you on a daily.